akka-docs/src/main/paradox/typed/fsm.md

```diff
@@ -0,0 +1,54 @@
+# Finite state machines
+
+With untyped actors there is explicit support for building @ref[Finite State Machines](../fsm.md). No support
+is needed in Akka typed as it more follows the [Erlang design principles](http://www.erlang.org/documentation/doc-4.8.2/doc/design_principles/fsm.html)
```

Comment:
mixed feelings on linking to erlang principles hm... On one hand, yes absolutely; on another, I wonder if it could scare people who are not "yet" in love with actors/erlang/akka hm...

Reply:
Agreed